assert/cmp/compare.go

```diff
@@ -248,7 +248,7 @@ type causer interface {
 }
 
 func formatErrorMessage(err error) string {
-	//nolint:errorlint // unwrapping is not appropriate here
+	// unwrapping is not appropriate here
```

Comment:
LOL looks like different versions of the linters disagree. Let me add `nolintlint` to the ignore list;


```
assert/cmp/compare.go:252:14: type assertion on error will fail on wrapped errors. Use errors.As to check for specific errors (errorlint)
        if _, ok := err.(causer); ok {
                    ^
```